btc/guardian.go

```
@@ -0,0 +1,672 @@
+	return tx.Deserialize(strings.NewReader(string(signedTxBytes)))
+}
+
+func (gc *GuardianClient) updateTx(prevTxID chainhash.Hash, updatedTx *wire.MsgTx, mergeTxs []*wire.MsgTx, values []int64) error {
+	var buf = new(bytes.Buffer)
+	updatedTx.Serialize(buf)
+	txHex := hex.EncodeToString(buf.Bytes())
+
+	mergeTxHexes := make([]string, len(mergeTxs))
+	for i, tx := range mergeTxs {
+		var mergeBuf = new(bytes.Buffer)
+		tx.Serialize(mergeBuf)
```

golangci-lint (1.62.2)

Error return value of `tx.Serialize` is not checked

(errcheck)